src/DevChatter.Bot.Core/Data/Model/QuizQuestion.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DevChatter.Bot.Core.Data.Model
+{
+    public class QuizQuestion : DataEntity
+    {
+        public string MainQuestion { get; set; }
+        public string Hint1 { get; set; }
+        public string Hint2 { get; set; }
+        public string CorrectAnswer { get; set; }
+        public string WrongAnswer1 { get; set; }
+        public string WrongAnswer2 { get; set; }
+        public string WrongAnswer3 { get; set; }
+
+        public Dictionary<char, string> LetterAssignment { get; private set; }
+
+        public string SetAndReturnLetterString()
```

Comment:
This method is taking in the answers, shuffling them around, and returning the formatted string which will be displayed in chat. Method name doesn't necessarily convey. Perhaps "GetRandomizedAnswers" or similar?